Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:22.04
+
+# bash as default
+SHELL ["/bin/bash", "-c"]
+
+# Common
+RUN apt update -y && apt install -y scons meson cmake python2-dev python3-dev vim-tiny git make g++
```

Comment:
I guess if this docker is using cmake then it doesn't need scons or meson

Reply:
That's true. Removed